code/index.html

```diff
@@ -2,17 +2,112 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width-device-width, initial-scale=1.0">
+    <title>Lolas little treasure box</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="Photos/favicon.ico" type="image/icon">
+   
   </head>
   <body>
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+      <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+      </div>
+      <img src="Photos/Logo.png" alt="Lola">
+      <nav>
+          <a href="#c1"> <strong>Das bin ich, <mark>Lola</mark></strong> </a>
```

Comment:
We should try to keep the styling to the CSS file, so here you could give a class name to the `a` tag and wrap Lola in a `span`, like this:
```
<a href="#c1" class="bold-text">
  Das bin ich, <span>Lola</span>
</a>
```
and then style accordingly in the CSS:
```
.bold-text {
  font-weight: bold;
}

span {
  background-color: yellow;
}
```

Reply:
PS. Super nice with the links to the sections ⭐

Reply:
Got it. The main problem I had was how to define slightly similar objects that were yet not quite the same - > that's why I put the style into the hml. Still struggling with that.

Reply:
You could always give them different class names if you need! And you could give them several class names, see the first example:
```
<p class="blue-text bold">Blue text</p>
<p class="green-text">Green text</p>
<p class="red-text">Blue text</p>

.blue-text {
  color: blue;
}

.green-text {
  color: green;
}

.red-text {
  color: red;
}

.bold {
  font-weight: bold;
}
```